backend/Dockerfile.driver

```diff
@@ -16,8 +16,8 @@
 ARG SOURCE_CODE=.
 
 # Use ubi8/nodejs-14 as base image
-#@follow_tag(registry.access.redhat.com/ubi8/go-toolset:1.20)
-FROM registry.access.redhat.com/ubi8/go-toolset:1.20 as builder
+#@follow_tag(registry.access.redhat.com/ubi8/go-toolset:1)
```

Comment:
Doesn't matter much but there is a typo here in the #@follow_tag, it should be `registry.access.redhat.com/ubi8/go-toolset:21` but apart from that tested overall and it looks good.

Reply:
Done, updated it to be `1.21`. Thanks!